### PR TITLE
Remove redundant encryption

### PR DESCRIPTION
**Pull request by jontours**
Need to refly pipeline then ready to go.

### CODE REVIEW

**Comment by jontours** [comment]
Pipeline is now reflied.

File: ci/pipeline.yml [hunk 1]
```diff
@@ -346,12 +345,10 @@ resources:
...
-  type: cg-common
+  type: s3-iam
   source:
-    bucket_name: ((secrets-bucket-production))
-    secrets_file: domain-broker-production.yml
-    secrets_passphrase: ((secrets-passphrase-production))
-    bosh_cert: bosh.pem
+    bucket: ((secrets-bucket-production))
+    versioned_file: domain-broker-production-decrypted.yml
     region: ((aws-region))
```
Comment (jmcarp) [inline ci/pipeline.yml]: Looks like this should be `region_name`: https://github.com/alphagov/paas-s3-resource#source-configuration

**Comment by jmcarp** [comment]
Looks good! Can you update the development secrets and drop the cg-common resource type too?

File: ci/pipeline.yml [hunk 2]
```diff
@@ -34,7 +33,7 @@ jobs:
       manifest: broker-src/bosh/manifest.yml
       vars_files:
       - terraform-secrets/terraform.yml
-      - common/secrets.yml
+      - common-development/domain-broker-development-decrypted.yml
```
[the remark this answers is not in the document]
Comment (jontours) [reply ci/pipeline.yml]: I used the pattern employed by @sharms.  It did help me understand the process, but now it's probably not needed.

[the remark this answers is not in the document]
Comment (jontours) [reply ci/pipeline.yml]: Files renamed and pipeline updated.

**Pull request merged**